utils.py

```diff
@@ -4,24 +4,61 @@
 import yaml
 import numpy as np
 from pyradiosky import SkyModel
+from os import environ
+from functools import cache
 
+HPC = environ.get('VALIDATION_SYSTEM_NAME')
+
+if HPC is None:
+    raise EnvironmentError(
+        "You must set the VALIDATION_SYSTEM_NAME environment variable to your system "
+        "name, corresponding to a file in hpc-configs."
+    )
```

Comment:
Should this step be add to the `README.md`?
Also, would this effectively prevent running the simulation locally?

Reply:
Yes, and yes. We could possibly add a `local` config to `hpc-configs/`? I don't foresee that we'd use it though...